code/game/objects/items/weapons/manuals.dm

```diff
@@ -113,88 +113,44 @@
+			D-1: 200 mols of nitrous oxide are released by the crystal.</br>
+			D-2: 200 mols of nitrogen are released by the crystal</br>
+			D-3: 250 mols of CO2 are released by the crystal</br></br></br>
 
-			1. Prepare collector arrays. This is done standard to any text on their function by wrenching them down, filling six plasma tanks with a plasma canister, and inserting the tank into the collectors one by one. Finally, initialize each collector.</br></br>
+			C Class: Events with mild effects to standard setups. Operator intervetion MAY be required. Engineering will be alerted on telecomms.</br></br>
+			C-1: 250 mols of oxygen are released by the crystal</br>
+			C-2: 250 mols of plasma are released by the crystal</br>
+			C-3: The temperature at which the SM starts to lose integrity is lowered for a few minutes.</br></br></br>
 
-			2. Prepare gas system. Before introducing any gas to the Supermatter engine room, it is important to remember the small but vital steps to preparing this section. First, set the input gas pump and output gas flow pump to 4500, or maximum flow. Second, switch the digital switching valve into the .up. position, in order to circulate the gas back toward the coolers and collectors.</br></br>
+			B Class: Events with significant effects to standard setups. SCRAM protocol may be required.</br></br>
+			B-1: The amount of plasma and O2 released by the SM is doubled for a few minutes.</br>
+			B-2: The amount of heat released by the SM is multiplied by 1.25 for a few minutes.</br>
+			B-1: The engine's EER is raised by 2000 for several minutes, regardless of outside factors.</br></br></br>
 
-			3. Apply N2 gas. Retrieve the two N2 canisters from storage and bring them to the engine room. Attach one of them to the input section of the engine gas system located next to the collectors. Keep it attached until the N2 pressure is low enough to turn the canister light red. Replace it with the second canister to keep N2 pressure at optimal levels.</br></br>
+			A Class: Events with SEVERE effects to standard setups. SCRAM protocol required.</br></br>
+			A-1: The engine's APC is shorted due to a power spike, leading to wires needing mending.</br>
+			A-2: The engine's air alarm sets its self to siphon mode due to radiological interference.</br>
+			A-3: The amount of plasma and O2 released by the SM is quadrupled for a few minutes.</br></br>
 
-			4. Begin primary emitter burst series. This means firing a single emitter for its first four shots. It is important to move to this step quickly. The onboard SMES units may not have enough power to run the emitters if left alone too long on-station. This engine can produce enough power on its own to run the entire station, ignoring the SMES units completely, and is wired to do so.</br></br>
+			S Class events: Events that require immediate intervention and a specialized responce to prevent a delamination event. Cordination with other departments is HIGHLY reccomended.</br></br>
```

Comment:
```suggestion
			S Class events: Events that require immediate intervention and a specialized response to prevent a delamination event. Coordination with other departments is HIGHLY recommended.</br></br>
```

Reply:
This was already commented on. TWICE! (its fixed)